src/modal/modal.scss

```diff
@@ -2,154 +2,194 @@
+  .iui-modal {
     left: 50%;
     top: 33%;
     transform: translate(-50%, -33%);
-    z-index: 1000;
     max-width: 50%;
     min-width: 380px;
     max-height: 100vh;
-    border-radius: $iui-border-radius;
-    box-shadow: $iui-elevation-24;
-    padding: $iui-baseline $iui-m;
-    box-sizing: border-box;
-
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
 
     @media screen and (max-width: 400px) {
       max-width: 95%;
       width: 95%;
       min-width: 95%;
     }
-
-    .iui-title-bar {
-      display: flex;
-      align-items: center;
-      margin-bottom: $iui-baseline;
-      justify-content: space-between;
-
-      > .iui-title {
-        font-size: $iui-font-size-subheading;
-        white-space: nowrap;
-        overflow: hidden;
-        text-overflow: ellipsis;
-        line-height: 1.5;
-      }
-    }
-
-    .iui-button-bar {
-      margin-top: $iui-baseline;
-      display: flex;
-      align-items: center;
-      justify-content: flex-end;
-
-      > .iui-button:not(:last-child) {
-        margin-right: $iui-s;
-      }
-    }
-
-    .iui-modal-content {
-      flex-grow: 2;
-      margin: 0 -#{$iui-m};
-      padding: 0 $iui-m;
-      overflow-y: auto;
-      overflow-y: overlay;
-    }
   }
 }
 
 @mixin iui-modal-full-page {
   $iui-full-page-modal-slide-in-speed: 0.3s;
   $iui-full-page-modal-slide-out-speed: 0.25s;
 
-  > .iui-modal-dialog {
-    display: flex;
-    flex-direction: column;
+  @media (prefers-reduced-motion: no-preference) {
+    transition: visibility $iui-speed-instant linear $iui-speed-slow, opacity $iui-speed ease-out $iui-speed-fast;
+  }
+
+  .iui-modal {
+    border-radius: 0;
     height: 100vh;
     width: 100vw;
-    left: 0;
-    top: 0;
-    transform: none;
+    inset: 0;
     max-width: initial;
     min-width: initial;
-    border-radius: 0;
+    display: flex;
+    flex-direction: column;
     will-change: transform;
-  }
-
-  // #region Full Page Modal Animations for CSS
-  @media (prefers-reduced-motion: no-preference) {
-    transition: visibility $iui-speed-instant linear $iui-speed-slow, opacity $iui-speed ease-out $iui-speed-fast;
-  }
-
-  > .iui-modal-dialog {
-    transform: translateY(100%);
     @media (prefers-reduced-motion: no-preference) {
+      transform: translateY(100%);
       transition: visibility $iui-speed-instant linear $iui-speed, opacity $iui-speed-instant linear $iui-speed,
         transform $iui-full-page-modal-slide-out-speed ease-in;
     }
   }
 
-  &.iui-modal-visible > .iui-modal-dialog {
+  &.iui-modal-visible .iui-modal {
     transform: translateY(0);
     @media (prefers-reduced-motion: no-preference) {
       transition: transform $iui-full-page-modal-slide-in-speed ease-out;
     }
   }
-  // #endregion Full Page Modal Animations for CSS
+}
+
+@mixin iui-modal-resizable {
+  background-color: transparent;
+  pointer-events: none;
+  z-index: 998;
+
+  .iui-modal {
+    pointer-events: initial;
```

Comment:
What it does?

Reply:
Because the parent (backdrop) has `pointer-events: none` we need to undo that for the actual dialog itself.
```scss
.iui-dialog-backdrop {
	pointer-events: none;

	.iui-dialog {
		pointer-events: initial;
	}
}
```